script/render.rb

```diff
@@ -25,9 +25,15 @@ def get_latest_release(bin)
   obj = JSON.parse(data)
   version = obj["version"]
   url = obj["assets"][bin]["url"]
+  # Parse the URL and add the utm parameter
+  uri = URI.parse(url)
+  new_query_ar = URI.decode_www_form(uri.query || '') << ["utm_source", "homebrew"]
```

Comment:
nitpick: can we use uppercase `HOMEBREW` for the utm value?

Reply:
no problem 👍